pkg/handler/discord/discord.go

```diff
@@ -776,3 +776,51 @@ func (h *handler) SweepOgifEvent(c *gin.Context) {
 
 	c.JSON(http.StatusOK, view.CreateResponse[any](nil, nil, nil, nil, "events swept successfully"))
 }
+
+// ListChannelMessageLogs godoc
+// @Summary Get list of messages in channel and its thread
+// @Description Get list of messages in channel and its thread
+// @id ListChannelMessageLogs
+// @Tags Discord
+// @Accept  json
+// @Produce  json
+// @Param discord_channel_id path string true "Channel Discord ID"
+// @Param startDate query string true "Start Date"
+// @Param endDate query string true "End Date"
+// @Success 200 {object} ListDiscordTextMessageLog
+// @Failure 400 {object} ErrorResponse
+// @Failure 404 {object} ErrorResponse
+// @Failure 500 {object} ErrorResponse
+// @Router /discords/channels/{discord_channel_id}/message-logs [get]
+func (h *handler) ListChannelMessageLogs(c *gin.Context) {
+	var input = request.GetChannelMessagesInput{
+		DiscordChannelID: c.Param("discord_channel_id"),
```

Comment:
@nnhuyhoang will this allow client call any discord_channel_id, if yes, gonna reject this for now

please follow up with me about this approach, need to validate/whitelist some channels only